helm/charts/oathkeeper/values.yaml

```diff
@@ -141,14 +141,19 @@ oathkeeper:
   managedAccessRules: true
 
 secret:
-  # -- set to true for this helm chart to manage the secret
-  # -- set to false for this helm chart to NOT manage the secret
-  # -- defaults to true
-  manage: true
-
-  # -- name of the secret to use. If empty, defaults to {{ include "oathkeeper.fullname" . }}
-  name: ""
+  # -- switch to false to prevent creating the secret
+  enabled: true
 
+  # ... and choose a different name for a secret you provide like this
```

Comment:
Please keep the comments consistent between charts ;) I have added suggestions in the first one